### ISSUE TITLE
Updating the cookie in hook instance A doesnt rerender hook instance B

### ISSUE THREAD

**Issue by breytex** [firatoezcan/useCookie#2]
When I `useCookie` multiple times with the same key in my project and call `setCookie` on one instance, the other instance doesn't re-render which is an unexpected behaviour.

You could implement a observer-pattern between all cookie instances to sync state with the same key.

**Comment by firatoezcan** [comment]
Good idea, but instead of an observer I'd rather switch to using the `onChanged` event for cookies as documented here: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/cookies/onChanged

Feel free to make a PR to add that functionality. This is not supported for Edge versions that are not Chromium though.
